frontend/index.html

```diff
@@ -46,7 +46,41 @@
         <!-- QRcode -->
         <script src="client/js/qrcode.js"></script>
 
-    <!-- This styling is for the language button-->    
+    <!-- Source i18n -->
+    <script src="js/lib/CLDRPluralRuleParser/CLDRPluralRuleParser.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.messagestore.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.fallbacks.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.language.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.parser.js"></script>
+    <script src="js/lib/jquery.i18n/jquery.i18n.emitter.js"></script>
+    <script src="js/global.js"></script>
```

Comment:
Better to import these from a CDN if possible so that the NREL cyber infrastructure can monitor for vulnerabilities.

Reply:
It will be done.